### DISCUSSION TITLE
（入站必看）一位传奇绳匠短暂的一生

### DISCUSSION THREAD

**Discussion by silverwolf510trillion** [share121/inter-knot#1052] [其他]
事情是这样的，中午我入驻了绳网，然后发了一张星见雅的图片，并且贴心地配上了b站作者的主页链接，然后我的绳网账号被标记为spam了（也就是被不允许上绳网了，应该是github的规矩，非会员不能贴链接），一代传奇绳匠就此落幕.......总之各位绳匠一定要注意这一点，不要随便贴链接，切记！（第一次吃到重视版权的苦头Orz）
![image](https://github.com/user-attachments/assets/d90dea5f-39a0-4a5e-a14d-0530982197cf)

**Comment by Qiling-XwX** [comment]
![87826858120198F683C834F51D99B578](https://github.com/user-attachments/assets/86bac432-b9cc-49df-8ee9-1474a073354c)

**Comment by MrPaiZi** [comment]
还有这种规矩？

**Comment by qiu-1214** [comment]
多谢提醒，还真是第一次知道

**Comment by silverwolf510trillion** [comment]
总之我现在注册了一个新的绳网账号，哈哈哈孩子们我回来了！

**Comment by silverwolf510trillion** [comment]
另外给新绳匠提醒一下，回复帖子后记得点一下unsubscribe，否则你的邮箱可能会被轰炸

**Comment by GanlinSama** [comment]
邮箱炸掉这个我记得绳网里有大佬给出过解决方法，一劳永逸的那种，可以去找找。
也就是今天下午三四点的帖子

**Reply by silverwolf510trillion** [reply]
现在绳网似乎还没有定时查找的功能，我现在看到的大多数中午那会的帖子都是分散出现的orz

**Comment by Buer-Nahida** [comment]
不会吧，Github允许打广告的
